helm/chart-operator/values.yaml

```diff
@@ -18,9 +18,22 @@ clusterDNSIP: 172.31.0.10
 deployment:
   requests:
     cpu: 50m
-    memory: 100Mi
+    memory: 256Mi
   limits:
-    memory: 350Mi
+    cpu: 100m
+    memory: 512Mi
+
+verticalPodAutoscaler:
+  enabled: true
+  maxAllowed:
+    cpu: 150m
```

Comment:
Is 150m enough on big clusters?

Reply:
Hmm, the previous max allowed was hard coded on the VPA to be 1 full core. Maybe we can increase it a little further, but I imagine the bottleneck on the memory.

Reply:
Yea, good point. I looked at some aggregated data about CPU requests across all our clusters. Let me adjust it.

Reply:
Updated as of: https://github.com/giantswarm/chart-operator/pull/1167/commits/3f3835f095d1afa8a4931006acd79cef37cf30a6